test/ng/compileSpec.js

```diff
@@ -294,6 +294,27 @@ describe('$compile', function() {
       inject(function($compile) {});
     });
 
+    it('should omit special chars before processing attribute directive name', function() {
```

Comment:
I would change "omit" to "ignore", because I think it bettr describes the behavior.